File: java_backup/my java/bishwarup_!/sum.java
 

class sum
{
public static void main(int n)
{
int i,j,f,t; double s=0,k;
f=1;k=0;t=-1;
for(i=1;i<=n;i++)
{
t=1;
k=0;
for(j=1;j<=i;j++)
{
k=k+(j*t);
t=t*-1;
}
s=s+(double)i/(double)k;
}
System.out.println("sum: "+s);
}
}

File: 4rth_Sem_c++_backup/PRACTICE/XAV/Jimut/sum.cpp
#include<iostream>
using namespace std;
class sum
{
	public:
		int sumno(int n)
		{
			int i,s=0;
			for (i=0;i<=n;i++)
	                	s=s+i;
			return s;

		}
};




int main()
{
	int i,n,s=0;
	system("clear");
	sum sum1;
	cout<<"\n Enter n(1-10) : "<<endl;
	cin >> n;
	//for (i=0;i<=n;i++)
	//	s=s+i;
	s = sum1.sumno(n);
	cout<<"Sum = "<<s;
	return 0;
}



